### DISCUSSION TITLE
FEAソフトウェアについて

### DISCUSSION THREAD

**Discussion by tkoyama010** [FEABeRjp/FEABeR#14] [General]
CoFEAプロジェクトのようにFEAソフトウェアの紹介と実行方法の説明が必要です。
https://cofea-ja.readthedocs.io/ja/latest/fea_software/calculix.html
https://cofea-ja.readthedocs.io/ja/latest/fea_software/code_aster.html
https://cofea-ja.readthedocs.io/ja/latest/fea_software/elmer.html

**Comment by mmer547** [comment]
用意します。

**Reply by tkoyama010** [reply]
ありがとうございます。GetFEMは私の方で準備します。

**Reply by tkoyama010** [reply]
FrontISTRも最近使用しているので協力できると思います。

**Reply by mmer547** [reply]
ありがとうございます。ちなみにですがRead the Docsのページ書くときはローカルで書いたページを確認する方法って、何かやられてますでしょうか？

**Reply by tkoyama010** [reply]
私はローカルでページをコンパイル・確認していません。Pull Requestを作成すると自動的にそのブランチのソースがCIでコンパイルされます。コンパイルエラーが出力されたら修正して再度CIを実行します。コンパイルエラーがなければ以下のように表示されます。 ``Details`` をクリックするとビルドされたページを確認することができます。レビューワーもビルドされたページを確認して変更が妥当か判断します。便利機能ですのでぜひお使いください。
![github-build-status-reporting](https://user-images.githubusercontent.com/7513610/146150349-4af53884-a1ff-4212-a46a-5f4c733e8644.gif)

**Reply by mmer547** [reply]
なるほど。ありがとうございます。

**Comment by mmer547** [comment]
将来的には広げていきたいですが、現状は下記ソフトを対象ですかね。

GetFEM
Code-Aster
CalculiX
FrontISTR

**Reply by tkoyama010** [reply]
それでよいと思います。

**Reply by tkoyama010** [reply]
FENICSxもどこまで対応できるのか確認したいです。

**Comment by mmer547** [comment]
CalculixですがPrePoMaxで使用している2.16と現行の2.19では同じインプットを実行できないケースがあるみたいです。他のソフトも互換性がなくなるときはあるので、バージョンの記載などを行い、バージョンの違うソフトは別のソフト扱いで進めた方がよいかと思います。
